### ISSUE TITLE
UI no longer fully visible on iOS

### ISSUE THREAD

**Issue by JohnW1ck** [hrydgard/ppsspp#8300]
Hey this happened a while back boefore the recent 2 updates and when i updated to the recent ppsspp dev-latest theres something wrong,i cant see anything other than homebrew,games,and recent thats it,i can still click on where the settings are and all but its all invisible,I am currently running ios 9.0.2 on iphone 5s Jailbroken.Plzz email me back if you get a chance to tell me whats wrong.
resendezraymond@gmail.com

**Comment by brujo5** [comment]
yeah the UI is totally broken in the Menu 

Could you please upload a screenshoot because I do not have my phone with me? thx

**Comment by unknownbrackets** [comment]
What's the first build it was broken in and the last build it worked in?

-[Unknown]

**Comment by tleple** [comment]
![image](https://cloud.githubusercontent.com/assets/7698892/11997303/67aadb44-aa5f-11e5-879b-517d8ccf9f2e.png)
![image](https://cloud.githubusercontent.com/assets/7698892/11997312/9abe55f6-aa5f-11e5-8487-34aafa2bf93e.png)
I have the same  issue in my device 
iOS 9.0.2 iPad Air

**Comment by unknownbrackets** [comment]
Maybe it's from this or something?  https://github.com/hrydgard/ppsspp/commit/13515bf23531ea767a522e6a55ea7dd11a3013bc#diff-175ee0143c02682123081bc19d223615L100

I suspected my recent UI layout changes, but they aren't merged yet.

-[Unknown]

**Comment by hrydgard** [comment]
Oh yeah, it could be that change, if that variable isn't set properly on iOS... will look into it later

**Comment by hrydgard** [comment]
That might help..

**Comment by JohnW1ck** [comment]
Ok so this happened 2 small updates before the one released recently,last build it worked on was when v1.1.1 was first released.I will upload a screenshot

**Comment by unknownbrackets** [comment]
It will take some time for the build to update, but I think that will fix it.

-[Unknown]

**Comment by JohnW1ck** [comment]
Ok here you go

**Comment by JohnW1ck** [comment]
Sorry im having trouble uploading a screen shot but the way it looks is exactly the way your screenshots look like.

**Comment by JohnW1ck** [comment]
Alright henrik rydgard when you can contact me please i have just one question and i dont mean to bug.
resendezraymond@gmail.com

**Comment by scorpio16v** [comment]
Yes, the bug is fixed in PPSSPP v1.1.1-440-g7eb662f. 
Tested on iPod Touch 6.Gen.

**Comment by brujo5** [comment]
UI fixed, but still crashes when play videos.

ppsspp 1.1.1 is the only that work pefect for me.

**Comment by brujo5** [comment]
UI fixed but now crashes at menu (after hit new game or start playing)

the crash in videos is fixed to

**Comment by JohnW1ck** [comment]
The crash for the video is fixed im really just waiting on news for the jit whether its going to be fixed soon.Thats all and im willing to pitch ideas maybe i can be a help to get jit to work on ios 9.0.2

**Comment by unknownbrackets** [comment]
We don't need multiple issues with people talking about Apple making life hard for their users.  That's #8122.  We already know it's happening, but it seems complex to fix.

This separate issue is fixed, so closing.

-[Unknown]
